haascli/stack.py

```diff
@@ -264,7 +264,7 @@ def events(ctx, stack_name):
                       (status,
                        event['ResourceType'],
                        event['Timestamp'].strftime('%Y.%m.%d-%X')),
-                    fg='red' if "FAILED" in status else 'black'))
+                    fg='red' if "FAILED" in status else 'white'))
```

Comment:
Hmm..this is interesting. Black is correct on my terminal. Meaning that "normal' fg color is terminal dependent.
Need to change this to an if statement.